src/Notepads/Strings/en-US/Resources.resw

```diff
@@ -429,10 +429,6 @@
     <value>selected</value>
     <comment>TextEditor: Singular form for the selected word count indicator.</comment>
   </data>
-  <data name="App_DragAndDrop_UIOverride_Caption_MoveTabHere" xml:space="preserve">
```

Comment:
Why you remove?

Reply:
I didn't even touch it, I don't know how it got removed. I am adding it back in.